addons/mail/models/mail_thread.py

```diff
@@ -4607,7 +4607,7 @@ def _message_update_content(self, message, body, attachment_ids=None, partner_id
         if body is not None:
             msg_values["body"] = (
                 # keep html if already Markup, otherwise escape
-                escape(body) + Markup("<span class='o-mail-Message-edited'/>")
+                escape(body) + Markup("<span class='o-mail-Message-edited' data-oe-expression='%s'/>") % fields.Datetime.to_string(fields.Datetime.now())
```

Comment:
Why couldn't this be a field? Why does it have to be put inside the body of the message? Seems counter-intuitive to set it up like this. Plus being put inside `data-oe-expression`, which is more akin to python expressions than dates, seems pretty meh.

Reply:
> Why couldn't this be a field?

Great question. It's **forbidden** to add any field on `mail.message`.
So we did this hack…

Reply:
We shouldn't add columns / fields to `mail.message` due to performance issues, so we have to find workarounds, one of which is to put the data in body like I made here.

`data-oe-expression` was the lesser badly named attributes that is currently whitelisted in our `html_sanitize`. @seb-odoo initiated some internal discussions on whitelisting more `data-`, but there's no final decision yet.